### PR TITLE
Test dev 1

### PR DESCRIPTION
**Pull request by Gorbachevaa**
Task DEV-1.

### CODE REVIEW

File: MultipleNumb.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
```
Comment (IgorKheidorov) [inline MultipleNumb.cs]: Remove unnecessary usings

File: MultipleNumb.cs [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+{
+    // Class that realize multiple numbers.
+    class MultipleNumb
+    {
+        // Method that checks multiplicity of the number 
+        public bool isMultiple(int a, int Mult)
+        {
+            return (a % Mult == 0);
+        }
+
```
Comment (IgorKheidorov) [inline MultipleNumb.cs]: Extra line

File: OutputOfNumb.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
```
Comment (IgorKheidorov) [inline OutputOfNumb.cs]: Remove unnecessary usings

File: Program.cs [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+using System.Threading.Tasks;
+
+namespace ConsoleApplication3
+{
+    class Program
+    {
+        // Program that outputs sequence 0 to 100, multiples of 3 output as 3*N.
+        static void Main(string[] args)
+        {
+            int AmountOfNumb = 100;
```
Comment (IgorKheidorov) [inline Program.cs]: Better use constants

File: MultipleNumb.cs (hunk 2 above)
Comment (Gorbachevaa) [inline MultipleNumb.cs]: Could you explain what is extra line? I don't understand.

**Pull request merged**